test/test.js

```diff
@@ -75,4 +75,20 @@ describe('gulp-potomo', function() {
        });
    });
 
+    it('5) Should compile PO to MO with msgfmt when path has a space.', function(done) {
+
+        gulp.src(path.join(__dirname, './fixtures/directory with spaces', 'en_GB.po'))
+            .pipe(potomo())
+            .pipe(sassert.length(1))
+            .pipe(gulp.dest(tmpFolder))
+            .pipe(sassert.end(function(){
+                var expected = fs.readFileSync(path.join(__dirname,'expected/en_GB.mo')).toString();
+                var actual = fs.readFileSync(path.join(__dirname, 'temp/en_GB.mo')).toString();
+                actual.should.equal(expected);
+                done();
+            }));
+
```

Comment:
can you remove the blank lines (90 and 91) and keep the space columns like the test number 4